workflowhelpers/internal/space.go

```diff
@@ -111,9 +120,16 @@ func (ts *TestSpace) Create() {
 }
 
 func (ts *TestSpace) Destroy() {
-	deleteOrg := internal.Cf(ts.CommandStarter, "delete-org", "-f", ts.organizationName)
-	EventuallyWithOffset(1, deleteOrg, ts.Timeout).Should(Exit(0))
-
+	if ts.isExistingOrganization {
+		setDefaultQuota := internal.Cf(ts.CommandStarter, "set-quota", ts.organizationName, "default")
```

Comment:
Why set the existing organization's quota when we delete a test space from within it?